example_input_files/loop.input

```diff
@@ -0,0 +1,42 @@
+# Input parameters for cvc-code
+# *******************************************************
+
+T =  64
+LX = 32
+LY = 32
+LZ = 32
+mu = 0.0030
+kappa = 0.1400645
+Nconf = 1240
+csw = 1.740000
+format = 0
+BCangleT = 1.
+BCangleX = 0
+BCangleY = 0
+BCangleZ = 0
+
+# prefix of source and propagator fiels,
```

Comment:
I guess this comment should still be changed to clarify what the code will build from these two prefixes...

Reply:
there was still this comment about sources and propagator prefixes